VMO - Using DataVolumes guide (#1918)
spectrocloud/librarium: docs/docs-content/vm-management/vm-management.md

Review [Create a VMO Profile](vm-packs-profiles/create-vmo-profile.md) and [Add Roles and Role Bindings](vm-packs-profiles/add-roles-and-role-bindings.md) to learn how to create the cluster profile and add roles and permissions that allow users to create and manage Virtual Machines (VMs). 

Palette VM Orchestrator provides various methods to quickly deploy VMs from out-of-the-box templates or from your organization's templates. To learn more about using and creating templates, review [Deploy VM From a Template](create-manage-vm/standard-vm-operations/deploy-vm-from-template.md) and [Create a VM Template](create-manage-vm/create-vm-template.md).

> While you can import template disks from external locations at VM deployment time, this approach is generally pretty slow. To learn about leveraging the `DataVolume` resource to import template disks once and create lightning-fast clones at VM deployment time, review [Using DataVolumes for disk templates](create-manage-vm/using-datavolumes.md).

Reviewer: ```suggestion
While you can import template disks from external locations at VM deployment time, this approach is not efficient. To learn about leveraging the `DataVolume` resource to import template disks once and create clones at VM deployment time, review the [Create disk templates](create-manage-vm/using-datavolumes.md) tutorial.
```

[Deploy VM From a Template](create-manage-vm/standard-vm-operations/deploy-vm-from-template.md) and
[Create a VM Template](create-manage-vm/create-vm-template.md).

> While you can import template disks from external locations at VM deployment time, this approach is not efficient. To
> learn about leveraging the `DataVolume` resource to import template disks once and create clones at VM deployment time,
> review the [Create disk templates](create-manage-vm/using-datavolumes.md) tutorial.

Reviewer 2: ```suggestion
While you can import template disks from external locations when deploying a VM, it can be inefficient. Instead, you can leverage the `DataVolume` resource to import template disks once and then clone them when deploying new VMs.
Review the [Create disk templates](create-manage-vm/using-datavolumes.md) guide for more information.
```

- [Create a VM Template](create-manage-vm/create-vm-template.md)

> - [Using DataVolumes for disk templates](create-manage-vm/using-datavolumes.md)

Reviewer: ```suggestion
- [Create disk templates](create-manage-vm/using-datavolumes.md)
```
